database/src/factories/login-user.factory.ts

```diff
@@ -0,0 +1,47 @@
+import Faker from 'faker'
+import { define } from 'typeorm-seeding'
+import { LoginUser } from '../../entity/LoginUser'
+import { randomBytes } from 'crypto'
+
+interface LoginUserContext {
+  email?: string
+  firstName?: string
+  lastName?: string
+  username?: string
+  description?: string
+  password?: BigInt
+  pubKey?: Buffer
+  privKey?: Buffer
+  emailHash?: Buffer
+  createdAt?: Date
+  emailChecked?: boolean
+  passphraseShown?: boolean
+  language?: string
+  disabled?: boolean
+  groupId?: number
+  publisherId?: number
+}
+
+define(LoginUser, (faker: typeof Faker, context?: LoginUserContext) => {
+  if (!context) context = {}
+
+  const user = new LoginUser()
+  user.email = context.email ? context.email : faker.internet.email()
+  user.firstName = context.firstName ? context.firstName : faker.name.firstName()
+  user.lastName = context.lastName ? context.lastName : faker.name.lastName()
+  user.username = context.username ? context.username : faker.internet.userName()
+  user.description = context.description ? context.description : faker.random.words(4)
+  user.password = context.password ? context.password : BigInt(0)
+  user.pubKey = context.pubKey ? context.pubKey : randomBytes(32)
+  user.privKey = context.privKey ? context.privKey : randomBytes(80)
+  user.emailHash = context.emailHash ? context.emailHash : randomBytes(32)
```

Comment:
this is required to validate the login password, I believe. Therefore a randomvalue might prevent the user to login